dll/win32/shell32/dialogs/view.cpp

```diff
@@ -860,21 +860,32 @@ ScanAdvancedSettings(SHELLSTATE *pSS, DWORD *pdwMask)
 }
 
 static BOOL CALLBACK
-RefreshBrowsersCallback(HWND hWnd, LPARAM msg)
+PostCabinetMessageCallback(HWND hWnd, LPARAM param)
 {
+    MSG &data = *(MSG*)param;
```

Comment:
Suggestion: move this inside the if-case where you're doing the PostMessage.

Reply:
It does not matter, there is no code executed here.